docs/reference/setup/jvm-options.asciidoc

```diff
@@ -58,8 +65,8 @@ a special syntax:
 
 * all other lines are rejected
 
-You can add custom JVM flags to this file and check this configuration into your
-version control system.
+You can add custom JVM flags to custom JVM options files and check these
```

Comment:
The repetition of `custom` bothers me here...but I'm struggling to think of something better. Maybe the writers can help.

Reply:
I removed the line, I don't think it adds anything useful to the docs.